Resources/doc/features.md

```diff
@@ -0,0 +1,12 @@
+Features
+===========
+
+Supports multiple entity managers 
+-------------
+
+Configure multiple entity managers according to http://symfony.com/doc/current/cookbook/doctrine/multiple_entity_managers.html
+
+Blamable behavior support
```

Comment:
Maybe move this to the annotations doc describing the Gedmo annotation w/ our addition of the use of headers to set the user.